### DISCUSSION TITLE
I made a GUI tool for testing HTTP/3

### DISCUSSION THREAD

**Discussion by alexandrehtrb** [microsoft/msquic#2568] [Show and tell]
Hello,

I got interested in HTTP/3 and QUIC a few months ago and noticed the lack of a GUI tool, like Postman, for testing. Postman does not support HTTP versions other than 1.1 currently.
I made a tool for that, with support for HTTP/2 and HTTP/3. It uses .NET 6 and msquic on Linux:

https://github.com/alexandrehtrb/Pororoca

Just sharing in case it could help someone!

**Comment by ManickaP** [comment]
This is great! I'm seeing that you're using our `HttpClient` for this. So if you hit any issues with H/3 let us know at https://github.com/dotnet/runtime.
